forte/data/data_store.py

```diff
@@ -759,6 +758,26 @@ def get_entry_index(self, tid: int) -> int:
             ]
         return index_id
 
+    def get_length(self, type_name: str) -> int:
+        r"""This function find the length of the `type_name` entry list.
+        It should not count None placeholders that appear in
+        non-annotation-like entry lists.
+
+        Args:
+            type_name (str): The fully qualified type name of a type.
+
+        Returns:
+            The count of not None entries.
+        """
+        if self._is_annotation(type_name):
+            return len(self.__elements[type_name])
+        else:
+            try:
+                delete_count = self.__deletion_count[type_name]
+            except KeyError:
+                delete_count = 0
```

Comment:
you can simply do `delete_count = self.__deletion_count.get(type_name, 0)`, and this whole branch is simply `return len(self.__elements[type_name] - self.__deletion_count.get(type_name, 0)`

Reply:
yes